### DISCUSSION TITLE
List user events with placement

### DISCUSSION THREAD

**Discussion by 5e0** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#299] [Q&A] [answered]
How to list a users' events and placements like this [https://fortnitetracker.com/profile/all/Ninja/events](https://fortnitetracker.com/profile/all/Ninja/events) ?

**Comment by LeleDerGrasshalmi** [comment, marked as answer]
That's not directly possible.

What you could do is fetch the full leaderboards for all events (tournament's) and then search for that player.

See events service in this docs repo
